can/interfaces/pcan/pcan.py

```diff
@@ -259,6 +264,14 @@ def __init__(
                     "Ignoring error. PCAN_ALLOW_ERROR_FRAMES is still unsupported by OSX Library PCANUSB v0.10"
                 )
 
+        if kwargs.get("auto_reset", False):
```

Comment:
Could you add the parameter to the `__init__()` method instead of using `kwargs.get`?
I do not know why some of the other parameters are not `__init__`-arguments but i thinks that's bad for readability.